stdlib/runtime/string.gr

```diff
@@ -562,12 +562,28 @@ toStringHelp = (grainValue, extraIndents, toplevel) => {
     let tag = grainValue & 7n
     if (tag == Tags._GRAIN_GENERIC_HEAP_TAG_TYPE) {
       heapValueToString(grainValue, extraIndents, toplevel)
-    } else if (tag == Tags._GRAIN_CHAR_TAG_TYPE) {
-      let string = usvToString(grainValue >> 8n)
-      if (toplevel) {
-        string
+    } else if (tag == Tags._GRAIN_SHORTVAL_TAG_TYPE) {
+      let shortVal = grainValue >> 8n
+      let shortValTag = (grainValue & 0xF8n) >> 3n
+      if (shortValTag == Tags._GRAIN_CHAR_SHORTVAL_TAG) {
+        let string = usvToString(shortVal)
+        if (toplevel) {
+          string
+        } else {
+          escapeChar(string)
+        }
+      } else if (
+        shortValTag == Tags._GRAIN_INT8_SHORTVAL_TAG ||
+        shortValTag == Tags._GRAIN_INT16_SHORTVAL_TAG
+      ) {
+        NumberUtils.itoa32(shortVal, 10n)
```

Comment:
Do these print with their suffix? I am actually realizing that I don't know if other types like this are printed with their suffixes

Reply:
No; the other types don't either (I think it's better without them imo)